content/_code-samples/nftoken-tester/js/nftoken-tester.html

```diff
@@ -210,14 +210,14 @@
 	await client.connect()
 	console.log("Connected to Sandbox")
 
-	const tokenID = tokenOfferIndex.value
-	const tokenIDs = [tokenID]
+	const tokenOfferID = tokenOfferIndex.value
+  const tokenOffers = [tokenOfferID]
```

Comment:
Indentation looks wrong here? I guess it's because the file originally uses tab characters rather than spaces.